inst/include/distributions/functors/normal_lpdf.hpp

```diff
@@ -11,22 +24,30 @@ namespace fims_distributions {
  */
 template<typename Type>
 struct NormalLPDF : public DensityComponentBase<Type> {
-    fims::Vector<Type> log_sd;
-    fims::Vector<Type> mu;
-    fims::Vector<Type> sd;
-    Type nll = 0.0;
-    std::vector<bool> is_na;
+    fims::Vector<Type> log_sd; /*!< log of the standard deviation of the distribution; can be a vector or scalar */
+    fims::Vector<Type> mu; /*!< mean of the distribution; can be a vector or scalar */
+    fims::Vector<Type> sd; /*!< standard deviation of the distribution; can be a vector or scalar */
+    Type nll = 0.0; /*!< total negative log-likelihood contribution of the distribution */
+    std::vector<bool> is_na; /*!< Boolean; if true, data observation is NA and the likelihood contribution is skipped */
     #ifdef TMB_MODEL
-    ::objective_function<Type> *of;
+    ::objective_function<Type> *of; /*!< Pointer to the TMB objective function */
     #endif
-    //data_indicator<tmbutils::vector<Type> , Type> keep;
+    //data_indicator<tmbutils::vector<Type> , Type> keep; /*!< total negative log-likelihood contribution of the distribution */
```

Comment:
I think this comment should be /*!< Indicator used in TMB one-step-ahead residual calculations */